Assets/Editor/Elements/SegmentedTrack.cs

```diff
@@ -0,0 +1,33 @@
+using UnityEngine.UIElements;
+
+public class SegmentedTrack : ITrack
```

Comment:
I don't think this is the right object to inherit from ITrack[Element?]. Perhaps only the right hand side element should have the manipulators and the like, that way it allows using it without the left hand side panel.